urllib3/response.py

```diff
@@ -169,6 +169,10 @@ def data(self):
     def connection(self):
         return self._connection
 
+    def info(self):
+        # Compatibility method for cookie handler
+        return self.headers
+
```

Comment:
Can you turn this comment into a docstring that also includes an admonition not to use this method?